tests/test_audio.py

```diff
@@ -28,14 +28,15 @@
 
 import telegram
 from tests.base import BaseTest, timeout
+from tests.reupload import get_file_id
 
 
 class AudioTest(BaseTest, unittest.TestCase):
     """This object represents Tests for Telegram Audio."""
 
     def setUp(self):
         self.audio_file = open('tests/data/telegram.mp3', 'rb')
-        self.audio_file_id = 'CQADAQADDwADHyP1B6PSPq2HjX8kAg'
+        self.audio_file_id = get_file_id('audio')
```

Comment:
Oh right... Whoops. Originally had it only do it when the class was imported (which would make it only run once in total over the entire lifetime of the tests). But along the way I realised that in some cases there's no need to upload all files if you for example are only running audio tests. But since it would have to way of figuring out which tests are gonna be run, I'll just switch it back to uploading on import I think. Unless you have a better idea? :)